plugins/ksp/src/org/jetbrains/kotlin/ksp/symbol/impl/java/KSTypeArgumentJavaImpl.kt

```diff
@@ -18,6 +18,10 @@ class KSTypeArgumentJavaImpl private constructor(val psi: PsiType) : KSTypeArgum
 
     override val origin = Origin.JAVA
 
+    override val location: Location by lazy {
+        TODO()
```

Comment:
Leaving the TODO in comments and returning a warning or simply `NonExistLocation` might be better.

Reply:
I have a feeling that current implementation of this class is not very right there therefore want to put it as a TODO() first. I can try to include the fix with this PR.

Reply:
Sure. Let me know when the PR is ready for review.

Reply:
I tried to use PsiTypeElement given the fact that PsiTypeElement correspond to actual source code element, but was unable to get everything work through (namely in other places where type are involved, like function return type, I can't find a way to get the PsiTypeElement but only PsiType).